provider/terraform/resources/resource_dataproc_cluster.go

```diff
@@ -364,7 +364,7 @@ func instanceConfigSchema() *schema.Schema {
 
 				// Note: preemptible workers don't support accelerators
 				"accelerators": {
-					// Terraform was unhappy when this was a set, even though it probably could be one
+					// Terraform throws an error if you try to set this field while it is a TypeSet.
```

Comment:
I still want to know why! Let's talk about it when we're both at our desks. Converting things from a List -> Set later is a breaking change, so if we can figure out a way to do it as a set now, that could avoid future headaches.